sdk/identity/identity/package.json

```diff
@@ -86,8 +86,8 @@
     "@azure/core-tracing": "^1.0.0",
     "@azure/core-util": "^1.11.0",
     "@azure/logger": "^1.0.0",
-    "@azure/msal-node": "^2.15.0",
-    "@azure/msal-browser": "^4.0.2",
+    "@azure/msal-node": "^3.2.1",
```

Comment:
I'm a _little_ concerned about doing a major version bump of `@azure/msal-node` in this release. Is it necessary for this PR?

Reply:
@maorleger Yes it has the fix needed for this PR. `3.2.1` version